attestation-container/protobuf/attestation-container.proto

```diff
@@ -17,4 +17,5 @@ message FetchAttestationRequest {
 
 message FetchAttestationReply {
   bytes attestation = 1;
+  bytes attestation_endorsement = 2;
```

Comment:
I think we should call this `attestation_endorsement_certificates` or similar, as we'll also add the certificate revocation list too, once supported by AMD?

Reply:
It makes sense to me

Reply:
[cd1c26d](https://github.com/microsoft/CCF/pull/4795/commits/cd1c26d0977ac6f14b808b0960d4271511cab7a8)